modules/apps/object/object-rest-impl/rest-openapi.yaml

```diff
@@ -478,6 +478,7 @@ paths:
             tags: ["ObjectEntry"]
     "/{objectEntryId}/object-actions/{objectActionName}":
         put:
+            operationId: putObjectEntryObjectActionObjectActionName
```

Comment:
Please, move this change to the previous commit where you include the endpoint in the `rest-openapi.yaml` file

Reply:
Done!